modules/local/spadeshybrid.nf

```diff
@@ -27,15 +15,15 @@ process SPADESHYBRID {
     path "${meta.id}_contigs.fasta.gz"                 , emit: contigs_gz
     path "${meta.id}_scaffolds.fasta.gz"               , emit: assembly_gz
     path "${meta.id}_graph.gfa.gz"                     , emit: graph
-    path '*.version.txt'                               , emit: version
+    path "versions.yml"                                , emit: versions
 
     script:
-    def software = getSoftwareName(task.process)
+    def spades_options = params.spades_options ?: ''
```

Comment:
See comment in `local/spades.nf`

Reply:
same as above, moved it so `modules.config`.